README.md

```diff
@@ -230,6 +230,17 @@ mgit exec 'echo `pwd`'
 # /home/mgit/packages/organization/repository-2
 ```
 
+
+### dump
```

Comment:
I'm not sure this is the right name. You dump data to some output, not save it and this is not data anyway. I'd go with something like `save-hashes`.